dockerfiles/startall.sh

```diff
@@ -51,6 +51,20 @@ dependency_check () {
   fi
 }
 
+dependency_update () {
+
+  if [ -z "$1" ]
+  then
+      CHART_SUBPATH="hpcc"
```

Comment:
minor: as no the hpcc helm chart has no dependencies, this function should prob. just error if nothing passed in.